### PR TITLE
Use diameter-dictionary module

### PR DESCRIPTION
**Pull request by rkovacevic**
Removes dictionary.json, and uses the dictionary from diameter-dictionary module.

### CODE REVIEW

**Comment by rkovacevic** [comment]
Also tested as a dependency from another project, e.g. 

```
  "dependencies": {
      "node-diameter": "https://github.com/node-diameter/node-diameter#pull/56/head"
  }
```

Seems to be working ok.

File: lib/diameter-dictionary.js [hunk 1]
```diff
@@ -2,10 +2,9 @@
 
 var _ = require('lodash');
 var fs = require('fs');
-var path = require('path');
 
 
-var dictionaryLocation = path.normalize(__dirname + '/../dictionary.json');
+var dictionaryLocation = require.resolve('diameter-dictionary');
```
Comment (atesgoral) [inline lib/diameter-dictionary.js]: Just `var dictionary = require('diameter-dictionary')` should work. `require` parses JSON modules by default.

**Comment by atesgoral** [comment]
Resolves #53

**Review by atesgoral — APPROVED** [review]
Looks good!

**Pull request merged**